pkg/features/volcano_features.go

```diff
@@ -43,6 +43,8 @@ const (
 
 	// ResourceTopology supports resources like cpu/memory topology aware.
 	ResourceTopology featuregate.Feature = "ResourceTopology"
+
```

Comment:
Can we add comment and reference documents to let users know the context of this feature gate?

Reply:
`SidecarContainers` is a kubernetes intree feature and has already a definition，and now we have imported other features like https://github.com/volcano-sh/volcano/blob/224ca98ab097d7788fc17ae5a6bbb0ba7a191653/pkg/scheduler/plugins/predicates/predicates.go#L291，so we doesn't need to re-define the feature.